dashboard/app/models/survey_result.rb

```diff
@@ -18,6 +18,8 @@
 class SurveyResult < ActiveRecord::Base
   include SerializedProperties
 
+  SYSTEM_DELETED = 'system_deleted'.freeze
```

Comment:
By "system_deleted" we just mean that a piece of code in our system was responsible for the deletion?